src/coreclr/nativeaot/System.Private.CoreLib/src/System/Reflection/Attribute.CoreRT.cs

```diff
@@ -0,0 +1,161 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection;
+using System.Diagnostics.CodeAnalysis;
+using System.Collections.Generic;
+using Internal.LowLevelLinq;
+using Internal.Reflection.Extensions.NonPortable;
+
+namespace System
+{
+    public abstract partial class Attribute
```

Comment:
This flipped the direction to match CoreCLR: `Attribute` has the actual implementation and `CustomAttributeExtensions` is the wrapper.